uvm/impl/com/untangle/uvm/SystemManagerImpl.java

```diff
@@ -124,9 +125,13 @@ protected SystemManagerImpl()
             logger.warn("No settings found - Initializing new settings.");
             this.setSettings(defaultSettings(), false);
         } else {
+            if(!StringUtils.isBlank(readSettings.getRadiusProxyPassword())){
+                readSettings.setRadiusProxyEncryptedPassword(PasswordUtil.getEncryptPassword(readSettings.getRadiusProxyPassword()));
+                readSettings.setRadiusProxyPassword(null);
+            }
             this.settings = readSettings;
 
-            if (this.settings.getVersion() < SETTINGS_VERSION) {
+            if (this.settings.getVersion() <= SETTINGS_VERSION) {
```

Comment:
Is this an intended change `<=` ?

Reply:
yes, Cause at time of restore the version would be same as SETTINGS_VERSION and encryption process will no excute. But I guess instead of updating the condition it would be better to update setting version.